Find a functionally equivalent code:android.graphics.Matrix.postTranslate:COMMENT
Method Modifier: public      
Comment:/**
 * Postconcats the matrix with the specified translation.
 * M' = T(dx, dy) * M
 */

@@ -1,3 +1,4 @@
 {
-    return native_postTranslate(native_instance, dx, dy);
+    native_postTranslate(native_instance, dx, dy);
+    return true;
 }

